data/update-lists.js

```diff
@@ -0,0 +1,53 @@
+const { execSync } = require("child_process");
+const readline = require("readline");
+const fs = require("fs");
+
+const rl = readline.createInterface({
```

Comment:
wouldn't it be easier to pass the key as a command parameter?

Reply:
make sense, done.